### PR TITLE
configure the develop environment

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <str_ruiling@outlook.com>

Close https://github.com/chaos-mesh/website/issues/310

### CODE REVIEW

**Comment by STRRL** [comment]
PTAL

/cc @YangKeao @Oreoxmt @g1eny0ung

File: docs/configure-development-environment.md [hunk 1]
```diff
@@ -28,59 +33,101 @@ After installing the above tools, follow the steps below to configure the toolch
...
-6. Compile Chaos Mesh:
+3. Install Chaos Mesh by Helm with Enabling Remote Debugging
 
    ```bash
-   make
+   helm upgrade --install chaos-mesh-debug ./helm/chaos-mesh --namespace=chaos-mesh-debug --create-namespace --set chaosDlv.enable=true --set controllerManager.replicaCount=1
    ```
 
-If no error occurs, you have successfully cnofigured the toolchain.
+   :::note We would set 3 replicas for `chaos-controller-manager` for HA purpose, `--set controllerManager.replicaCount=1` would create 1 instance of `chaos-controller-manager` for easier debugging. :::
```
Comment (g1eny0ung) [inline docs/configure-development-environment.md:108]:
```suggestion
   :::note

   We would set 3 replicas for `chaos-controller-manager` for HA purpose, `--set controllerManager.replicaCount=1` would create 1 instance of `chaos-controller-manager` for easier debugging.

   :::
```

File: docs/configure-development-environment.md [hunk 2]
```diff
@@ -28,59 +33,101 @@ After installing the above tools, follow the steps below to configure the toolch
...
+   kubectl -n chaos-mesh-debug port-forward chaos-controller-manager-766dc8488d-7n5bq 58000:8000
+   ```
 
-```bash
-kind delete cluster --name=kind
-```
+   Then we could access the remote delve debugger server with `127.0.0.1:58000`.
 
-To start Chaos Dashboard, run the following command:
+   :::note We always use `8000` in the pod for serving the delve debug server, that's a convention. You could find that in helm templates files. :::
```
Comment (g1eny0ung) [inline docs/configure-development-environment.md:122]:
```suggestion
   :::note

   We always use `8000` in the pod for serving the delve debug server, that's a convention. You could find that in helm templates files.

   :::
```

**Review by g1eny0ung — APPROVED** [review]
Rest LGTM.

I think I can supplement **Debug Chaos Dashboard in local environment** after this PR is merged.

**Review by YangKeao — APPROVED** [review]
LGTM

**Pull request merged**